overlord/assertstate/assertstate.go

```diff
@@ -765,12 +765,12 @@ func validationSetAssertionForEnforce(st *state.State, accountID, name string, s
 	return vs, latest, err
 }
 
-// TryEnforceValidationSets tries to fetch the given validation sets and
+// TryApplyEnforcedValidationSets tries to fetch the given validation sets and
 // enforce them (together with currently tracked validation sets) against
 // installed snaps, but doesn't update tracking information in case of an error.
 // It may return snapasserts.ValidationSetsValidationError which can be used to
 // install/remove snaps as required to satisfy validation sets constraints.
-func TryEnforceValidationSets(st *state.State, validationSets []string, userID int, snaps []*snapasserts.InstalledSnap, ignoreValidation map[string]bool) error {
+func TryApplyEnforcedValidationSets(st *state.State, validationSets []string, userID int, snaps []*snapasserts.InstalledSnap, ignoreValidation map[string]bool) error {
```

Comment:
sorry, I should have thought through my suggestion, TryApply is not quite grammatical, it would be TryApplying, but we can probably simplify to s/TryApplyEnforced.../TryEnforced.../